### DISCUSSION TITLE
Troubles with Smart Collection Creation

### DISCUSSION THREAD

**Discussion by JohnDoranNY** [meisnate12/Plex-Meta-Manager#1521] [Q&A]
I am trying to create a Top 250 list.. simple enough.  It runs fine up until the time to create the collection...

=========================== Updating Details of IMDb Top 250 Collection ============================|
[2023-07-18 13:56:20,370] [builder.py:2873]           [INFO]     |                                                                                                    |
[2023-07-18 13:56:20,376] [plex_meta_manager.py:833]  [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "C:\Users\johnd\Plex-Meta-Manager\plex_meta_manager.py", line 779, in run_collection        |
                                                                 |     details_list = builder.update_details()                                                        |
                                                                 |   File "C:\Users\johnd\Plex-Meta-Manager\modules\builder.py", line 2915, in update_details         |
                                                                 |     self.obj.editSummary(summary[1])                                                               |
                                                                 |   File "C:\Python\v09\lib\site-packages\plexapi\mixins.py", line 668, in editSummary               |
                                                                 |     return self.editField('summary', summary, locked=locked)                                       |
                                                                 |   File "C:\Python\v09\lib\site-packages\plexapi\mixins.py", line 557, in editField                 |
                                                                 |     return self._edit(**edits)                                                                     |
                                                                 |   File "C:\Python\v09\lib\site-packages\plexapi\base.py", line 552, in _edit                       |
                                                                 |     self._server.query(part, method=self._server._session.put)                                     |
                                                                 |   File "C:\Python\v09\lib\site-packages\plexapi\server.py", line 753, in query                     |
                                                                 |     raise NotFound(message)                                                                        |
                                                                 | plexapi.exceptions.NotFound: (404) not_found; (redacted)/library/sections/1/all?id=184606&summary.locked=1&summary.value=The%20%2ATop%20250%2A%20Movies%20of%20all%20time%20as%20ranked%20by%20IMDb.&type=18 <html><head><title>Not Found</title></head><body><h1>404 Not Found</h1></body></html> |
                                                                 |                                                                                                    |
[2023-07-18 13:56:20,379] [plex_meta_manager.py:834]  [ERROR]    | Unknown Error: (404) not_found; (redacted)/library/sections/1/all?id=184606&summary.locked=1&summary.value=The%20%2ATop%20250%2A%20Movies%20of%20all%20time%20as%20ranked%20by%20IMDb.&type=18 <html><head><title>Not Found</title></head><body><h1>404 Not Found</h1></body></html> |

I manually searched and the label was updated for all the movies in the list but no collection is created.  I optimized db, reset plex, etc but always the same issue trying to create the collection... anyone have an idea what is wrong?

**Comment by chazlarson** [comment]
There have been changes on the IMDB side that break the IMDB charts, among other things.  Once those changes settle down, changes will be made in PMM to account.  For time being, the imdb charts don't work.
